test/collection_test.js

```diff
@@ -420,3 +420,15 @@ describe("Collection", () => {
     });
   });
 });
+
+describe("computeSnapshotAt()", () => {
+  // Note: most of the tests covering this API are done in integration_test.js
+
+  it("should raise when a snapshot records timestamp exceeds requirements", () => {
+    const records = [{id: 1, last_modified: 43}, {id: 2, last_modified: 44}];
+    const changes = [{action: "create", target: {data: {last_modified: 40}}}];
```

Comment:
If we want to simulate that the `history` plugin was enabled after the record `id: 1` was created, then we should have `last_modified: 44` (or 45, but above 43)

Imagine the following scenario:

- `id: 1` is created (ts: 43)
- plugin is enabled 
- `id: 2`is created (ts: 44), change entry `action: create` is tracked (ts >= 44)